decidim-dev/lib/decidim/dev/test/rspec_support/webmock.rb

```diff
@@ -0,0 +1,2 @@
+require "webmock/rspec"
+WebMock.allow_net_connect!
```

Comment:
As discussed offline we used this because we are using a custom host for each organization.